### PR TITLE
Initial page

### PR DESCRIPTION
**Pull request by paulsaenzsucre**
In this PR I implement the following features:

📌 Create DOM elements dynamically for the tasks array.
📌 Style the DOM elements to show the tasks like a to-do list and show their properties in a beautiful way.

Also, I did the following activities:

📌 I did setup my local environment with linters and webpack.
📌 I coded a function to show the tasks array on the screen using the DOM.
📌 I used webpack to automate repetitive task and to build my project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,163 @@
...
+
+- Code editor like: [Visual Studio code](https://code.visualstudio.com/), [Sublime text](https://www.sublimetext.com/) or others...
+
+- Npm and Node.js
+
+### Setup
+
+- You can download the **Zip** file on the GitHub repository, or clone the repository with:
+```
+$git clone https://github.com/elafildecolor/MyPortfolio.git
```
Comment (GzimAsani) [inline README.md]: - [x] Update your Readme with information about this project

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello  @elafildecolor                                 :wave:   
 Good job so far! There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job with webpack and all, your project meets the requirements  :+1: 

 ## Required Changes 
- [ ] You have met the project requirements except for readme file, kindly update that with regards to the current project: check the comment below :arrow_double_down: 



 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,163 @@
...
+### Setup
+
+- You can download the **Zip** file on the GitHub repository, or clone the repository with:
+```
+git clone https://github.com/elafildecolor/To-Do-List.git
+```
+
+- Access the cloned directory with:
+```
+cd MyPortfolio
```
Comment (1GORDON) [inline README.md]: Great job @elafildecolor 🤝

- [x] [Optional] You may kindly update this line as well by replacing `MyPortfolio` with `To-Do-List`.

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @elafildecolor ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**